tests/pact/lists.repl

```diff
@@ -0,0 +1,63 @@
+; make-list tests
+(expect
+ "make-list: Non-positive lengths return empty lists"
+ (fold
+  (and)
+  true
+  (map (= []) (list (make-list -1 true) (make-list 0 true) (make-list -100 true))))
+ true)
+
+(expect
+ "make-list: Simple sanity test"
+ (make-list 5 "hello")
+ ["hello" "hello" "hello" "hello" "hello"])
+
+; enumerate tests
+
+(expect
+ "enumerate: Enumerate numbers from 0 to 10."
+ (enumerate 0 10)
+ [0 1 2 3 4 5 6 7 8 9 10])
```

Comment:
Doh! You've reversed `expected` and `actual` in all of these :) please fix

Reply:
`transpose-sexps` to the rescue